doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -1333,6 +1342,11 @@ The following ruleset removes any query parameters set by the client.::
    cond %{REMAP_PSEUDO_HOOK}
    rm-destination QUERY
 
+Remove only a few select query parameters:
+
+   cond %{REMAP_PSEUDO_HOOK}
+   rm-destination QUERY foo,bar
+
```

Comment:
Perhaps also

```

Keep only a few select query parameters -- removing the rest:

   cond %{REMAP_PSEUDO_HOOK}
   rm-destination QUERY foo,bar [I]
```   
(if that is indeed how to do it)

Reply:
Added, thanks!